src/libs/fsdrv/lv_fsdrv.h

```diff
@@ -19,6 +19,8 @@ extern "C" {
  *      DEFINES
  *********************/
 
+#define MAX_PATH_LEN 256
```

Comment:
`LV_FS_MAX_PATH_LEN`

For public interfaces exposed to the outside, please add namespaces to avoid naming pollution.

Reply:
Done